MAVProxy/tools/MAVExplorer.py

```diff
@@ -332,16 +333,22 @@ def load_graphs():
     mestate.graphs = []
     gfiles = ['mavgraphs.xml']
     for dirname, dirnames, filenames in os.walk(mp_util.dot_mavproxy()):
+        # Skip XML files in the LogMessages subfolder
+        if os.path.basename(dirname) == "LogMessages":
+            continue
         for filename in filenames:
             if filename.lower().endswith('.xml'):
                 gfiles.append(os.path.join(dirname, filename))
 
+    # list of parameter files to be skipped when loading graph XML
+    paramfiles = ["ArduSub.xml", "ArduPlane.xml", "APMrover2.xml", "ArduCopter.xml", "AntennaTracker.xml", "Blimp.xml", "Rover.xml"]
+    # loop though files
     for file in gfiles:
         if not os.path.exists(file):
             continue
         # skip parameter files.  They specify an encoding, and under
         # Python3 this leads to a warning from etree
-        if os.path.basename(file) in ["ArduSub.xml", "ArduPlane.xml", "APMrover2.xml", "ArduCopter.xml", "AntennaTracker.xml", "Blimp.xml", "Rover.xml"]:
```

Comment:
This patch is no longer required.  It could also be a `frozenset` in place so keeping the thing tightly scoped while increasing efficiency - but really, just remove this bit :-)

Reply:
removed change.